packages/webdriverio/src/commands/element/click.ts

```diff
@@ -111,6 +133,10 @@ async function elementClick(element: WebdriverIO.Element) {
     try {
         return await element.elementClick(element.elementId)
     } catch (error) {
+        // We will never reach this code in the case of a mobile app, the implicitWait will wait for the element but will throw
```

Comment:
Let's add a check for mobile and assign the never type to it so that it is clear this code should never be reached in this particular scenario

Reply:
Not sure if I understand what you mean here, can you give some more context?

Reply:
@wswebcreation can you clarify why this catch block will never be reached for mobile apps?